packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+ *
+ * This module provides utility functions used by Auto Logging to work with surfaces
+ *
+ * @flow strict-local
+ * @format
+ * @oncall am_logging
+ */
+import {addEventListener, removeEventListener} from "@hyperion/hyperion-dom/src/IEventTarget";
```

Comment:
it is better to do `import * as IEventTarget from "..."` so that these functions have more meaning when used in the code.